Documentation/ProjectFile/properties/property/CapillaryPressureVanGenuchten/c_CapillaryPressureVanGenuchten.md

```diff
@@ -0,0 +1 @@
+\copydoc MaterialPropertyLib::SaturationVanGenuchten
```

Comment:
should be pointing to capillary pressure.

Reply:
They are same by the same \copdoc. Anyway replaced it with CapillaryPressureVanGenuchten.

Reply:
Not yet everywhere.

Reply:
Yes, only class documentation was changed because the class members of   `_residual_saturation`,  ` _maximuml_saturation` are not directly taken from the parameter input, which are set as `residual_liquid_saturation`, `residual_gas_saturation` for the consistency with `createSaturationVanGenuchten`.
In the recent changes, I kept the copying the documentations these two parameters from that for  `createSaturationVanGenuchten`, and made the  the documentations of other three parameters be copied from the class.

By the way, I suggest to replace `residual_gas_saturation` of capillary pressure classes and saturation classes with `maximum_liquid_saturation` in future. Because for these models, saturation means liquid saturation, we do not need highlight  `residual_gas_saturation` there.

Reply:
The problem with this approach of referring to the same documentation from different classes is that it creates dependencies which are not there. If the SaturationVanGenuchten is renamed/moved what ever, how many places do you need to change?
Keep things simple. There is no need to create any dependencies in such simple implementations.

Reply:
Documented the left two parameters directly (see 113fa68).